tests/ekore/anomalous_dimensions/unpolarized/time_like/test_init.py

```diff
@@ -1,10 +0,0 @@
-import pytest
-
-import ekore.anomalous_dimensions.unpolarized.time_like as ad_ut
-
-
-def test_init():
```

Comment:
don't simply delete - replace with something real

Reply:
looking at the analogous tests in space-like case, I understand the ones which check the dimensions of the output, however the others are not fully clear to me. perhaps we could discuss a little about this over zoom?